bin/gwdiff/gwdiff.ml

```diff
@@ -135,13 +135,13 @@ let dmy_to_sdn_range_l dmy =
           {
             year =
               (if dmy.month = 0 || (dmy.month = 12 && dmy.day = 0) then
-               dmy.year + 1
-              else dmy.year);
+                 dmy.year + 1
+               else dmy.year);
             month =
               (if dmy.month = 0 then 1
-              else if dmy.day = 0 then
-                if dmy.month = 12 then 1 else dmy.month + 1
-              else dmy.month);
+               else if dmy.day = 0 then
+                 if dmy.month = 12 then 1 else dmy.month + 1
+               else dmy.month);
             day = (if dmy.day = 0 then 1 else dmy.day);
             prec = (if dmy.month = 0 || dmy.day = 0 then Before else Sure);
```

Comment:
I don't want to look futher into what this code does for now but we had issues with dates and their encoding with day=0 for unknown day.
So the buggyness of this code looks high to me, maybe it should use a function for the Date module?

Also maybe gwdiff should be in geneweb-contrib?

Reply:
not my code!!

Reply:
I know!!
but it was touched because of the format

let's not worry about this code at all